### ISSUE TITLE
[HOLD for payment 2024-03-18] [$500] [Simplified Collect][Members] WorkspaceMemberDetailsPage: full page implementation (without the Change owner feature)

### ISSUE THREAD

**Issue by luacmartins** [Expensify/App#35715]
Implement `WorkspaceMemberDetailsPage: full page implementation (without the Change owner feature)` from the [Simplified Collect: Profile, Members, Categories design doc](https://docs.google.com/document/d/1gk3xqOs7epMbUrSSiX8K7YcqfPLVgqEos0sf-D-GMDA/edit)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01c86d95485fe71541</li>
        <li>Upwork Job ID: 1755658986412068864</li>
        <li>Last Price Increase: 2024-02-08</li>
    </ul>
</details>

**Comment by luacmartins** [comment]
@burczu can you please comment here so I can assign you to the issue?

**Comment by burczu** [comment]
Hi! I'm Bartek from Callstack - expert contributor group. I'll work on this issue so please assign me. Thank you.

**Comment by laurenreidexpensify** [comment]
@burczu is there a draft PR up for this yet?

**Comment by burczu** [comment]
@laurenreidexpensify this is one of the issues regardning bigger project and I'm working on these issues in an order - I didn't start working on this issue yet - I'll let you know here once started

**Comment by abdulrahuman5196** [comment]
Not overdue. Waiting on PR

**Comment by trjExpensify** [comment]
Hehe, looks like @burczu is ⭐'d on the issue, so only his updates will clear the `overdue` label and keep Melvin at bay.

**Comment by burczu** [comment]
not overdue - I haven't started the implementation yet

**Comment by abdulrahuman5196** [comment]
Waiting on PR

**Comment by luacmartins** [comment]
@burczu is this on hold for another issue?

**Comment by burczu** [comment]
yes, I'll write a comment here once I start the implementation

**Comment by abdulrahuman5196** [comment]
Waiting on PR

**Comment by CortneyOfstad** [comment]
@burczu any update on the implementtion?

**Comment by burczu** [comment]
Still not started working on this issue.

**Comment by CortneyOfstad** [comment]
@burczu can you have a PR created by EOW? This has been assigned for a few weeks, so we would like to get this resolved as soon as we can. Thank you!

**Comment by burczu** [comment]
@CortneyOfstad I'm not sure if you are aware that this issue is one of 7 issues related to the new Members page (this is part of the Simplified collect project). I'm assigned to all of them and work on them one by one - I know this was assigned some time ago but this was intended.

Good news is that this issue is next in my queue and I'll start working on it once this PR is merged: https://github.com/Expensify/App/pull/37199 - most likely tomorrow. But I don't think the PR for this issue will be ready EOW - I think Monday/Tuesday is more probable.

**Comment by luacmartins** [comment]
@CortneyOfstad thanks for checking in and asking for an update. As @burczu said, they are making progress on the other issues in the pipeline. We'll get to this one soon enough :)

**Comment by CortneyOfstad** [comment]
@burczu no worries and thank you for that additional context! I was not aware of that, so I recommend you sharing that now!

**Comment by burczu** [comment]
Just to inform everyone: I'm starting the implementation of this issue and will post some daily updates here on the progress.

**Comment by luacmartins** [comment]
@burczu do we have a draft PR that you can link to this issue yet?

**Comment by burczu** [comment]
> @burczu do we have a draft PR that you can link to this issue yet?

Added

**Comment by luacmartins** [comment]
Thanks for the update!

**Comment by burczu** [comment]
@luacmartins The PR is ready for review.

**Comment by luacmartins** [comment]
Left a review on the draft PR today

**Comment by laurenreidexpensify** [comment]
@abdulrahuman5196 please apply to job in Upwork - https://www.upwork.com/jobs/~01183d516188835326- thanks

**Comment by abdulrahuman5196** [comment]
> @abdulrahuman5196 please apply to job in Upwork - https://www.upwork.com/jobs/~01183d516188835326- thanks

@laurenreidexpensify This link shows 404 error for me.

**Comment by laurenreidexpensify** [comment]
Sorry @abdulrahuman5196 https://www.upwork.com/jobs/~01183d516188835326

**Comment by abdulrahuman5196** [comment]
> Sorry @abdulrahuman5196 https://www.upwork.com/jobs/~01183d516188835326

@laurenreidexpensify Applied on upwork.

**Comment by laurenreidexpensify** [comment]
Thanks - offer sent @abdulrahuman5196, will issue payment once you accept

**Comment by laurenreidexpensify** [comment]
- Contributor: @burczu is from an agency-contributor and not due payment
- ROLE: @abdulrahuman5196 paid $500 via Upwork
